sbi/density_estimators/flow.py

```diff
@@ -0,0 +1,53 @@
+from pyknos.nflows import flows
+from sbi.density_estimators import DensityEstimator
+import torch
+from torch import Tensor, nn
+from typing import Optional,Union
+from warnings import warn
+
+
+class FlowDensityEstimator(DensityEstimator):
```

Comment:
I would make this very explicit here and call it `NFlowsNSF`, `NFlowsMAF`, ...

Reply:
What is the motivation for this? For all NFlow flows, the `.log_prob()` and `.sample()` syntax is the same, and the loss is the same in all cases as well. So why split based on the type of flow?